src/common/templates/includes/macros.njk

```diff
@@ -16,6 +16,12 @@
 	{% endfor %}
 {% endmacro %}
 
+{% macro empty_table_message(message) %}
```

Comment:
WAVE mentions a piece of text that part of a table but not a cell should be marked as a caption or somesuch.

This is now the case, with styling to set it to the bottom, which looks ok to be honest. Not sure what effect this might have on newer browsers (was not aware of a caption tag before).